packages/amplify-container-hosting/lib/ElasticContainer/index.js

```diff
@@ -228,7 +228,7 @@ export async function generateHostingResources(
 
   const projectBackendDirPath = context.amplify.pathManager.getBackendDirPath();
 
-  /** @type {import('amplify-category-api').ApiResource & {service: string, domain: string, providerPlugin:string, hostedZoneId: string, iamAccessUnavailable: boolean}} */
+  /** @type {import('@aws-amplify/amplify-category-api').ApiResource & {service: string, domain: string, providerPlugin:string, hostedZoneId: string, iamAccessUnavailable: boolean}} */
```

Comment:
remove?

Reply:
Why? The package name is now `@aws-amplify/amplify-category-api`

Reply:
Oh I see, this is some sort of comment to resolve types in js files. nvm